bundle/config/variable/lookup.go

```diff
@@ -0,0 +1,277 @@
+// Code generated from OpenAPI specs by Databricks SDK Generator. DO NOT EDIT.
+
+package variable
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/databricks/databricks-sdk-go"
+)
+
+type Lookup struct {
+	Alert string `json:"alert,omitempty"`
+
+	ClusterPolicy string `json:"cluster-policy,omitempty"`
```

Comment:
We should use lower case for these field names to stay consistent with the rest of the config.